docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.3"
+
+services:
+  need_pedia:
+    build:
+      context: ../../.rvm/gems/ruby-2.6.3/gems/popper_js-1.16.0/assets/javascripts
+      dockerfile: /home/clicktechlabs/workspace/needpedia/config/Dockerfile
```

Comment:
in your changes this line was as below
`dockerfile: Dockerfile`
so I had to provide the path like for it to work